app/views/finders/show.atom.builder

```diff
@@ -0,0 +1,21 @@
+atom_feed do |feed|
+  feed.title(@finder.name)
+
+  dates = []
+  @results.results.map(&:metadata).each do |r|
```

Comment:
The update logic is a bit heavy for a template(ish), might be worth moving to the presenter? Would have the bonus of making it more easily testable too.